build.sh

```diff
@@ -118,3 +119,6 @@ if [ "$1" == "uninstall" ]; then
 else
   install_csq
 fi
+
+# For making Csq API universal
+sudo cp -r Core/Runtime /usr/include/Csq/
```

Comment:
you should put it in install function and in uninstall function you need to remove that